readthedocs/profiles/views.py

```diff
@@ -169,3 +174,35 @@ class TokenDeleteView(TokenMixin, DeleteView):
 
     def get_object(self, queryset=None):  # noqa
         return self.request.user.auth_token
+
+
+class UserSecurityLogView(PrivateViewMixin, ListView):
+    model = AuditLog
+    template_name = 'profiles/private/security_log.html'
+    days_limit = 90
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['days_limit'] = self.days_limit
+        context['filter'] = self.filter
+        return context
+
+    def get_queryset(self):
+        user = self.request.user
+        days_ago = timezone.now() - timedelta(days=self.days_limit)
+        queryset = AuditLog.objects.filter(
+            user=user,
+            action=AuditLog.AUTHN,
```

Comment:
Will we support the other auditlog entry types in this UI? If so I'll eventually want this field as a field in the filter instance, similar to build/project filters.